index.md

```diff
@@ -1,4 +1,44 @@
 ---
 layout: docs
-title:  "Using our Collections API"
----
\ No newline at end of file
+title:  "Getting Started with our Collections API"
+---
+
+## What is an API? 
+
+An API (Application Programming Interface) is a set of instructions that tells one software application how to "talk to", or transmit data, to another. If you have used [DIMES](https://dimes.rockarch.org/), our site to enable search and access to our collections, you have seen our API in action. For example, when you initiate a search for something in DIMES, your browser makes a request to the API, which delivers the data as the search results. 
+
+With the API, you can bypass the website interface and instead send requests directly to get the data. With this direct access, you can explore the data in different ways and build new things.
+
+## Who can use our API? 
+
+Anyone can use it! We have made it publicly available for people like you to freely access. There is no sign up or authentication required.
+
+### What is the data and where does it come from? 
+
+Our archival holdings are arranged and described by Rockefeller Archive Center archivists to make them accessible and facilitate use by providing information about their content and context. Archivists create descriptive information about: 
+
+1. Archival records.
+2. Agents, who are records creators and other people, organizations, and families related to the records.
+3. Relevant biographical, historical, or administrative activities.
+4. The relationships between these records, agents and activities.
+
+All of this information is managed in its own system and made available via the API. The API also includes data used specifically in the DIMES website to improve search and discovery on the site.
+
+Learn more about what data is available by [browsing the available API endpoints](https://api.rockarch.org), or points of entry to communicate with the API.
+
+### Licensing 
+
+This data is released under a Creative Commons Zero (CC0) public domain dedication. See our [full licensing statement](https://docs.rockarch.org/archival-description-license/) for specific license terms as well as best practices for repurposing this data.
+
+## What can I do with the API? 
+
+You can use the API to get our collections data, and you can use that data in any way you like. Build applications, visualize and analyze data in new ways, search and browse, and generally make new connections. 
```

Comment:
"Build applications, visualize and analyze data in new ways, search and browse, and generally make new connections." is an incomplete sentence - should it be combined with the preceding sentence?